INSTALL.md

```diff
@@ -41,9 +38,9 @@ are checked at configure time, preventing compilation. In the,
 unlikely, event an incompatibility remains undetected, using Floats
 would enable to prove False on this architecture.
```

Comment:
Perhaps you could update the wording of this paragraph?  There are a couple
very apparent errors.  I suggest:

Primitive floating-point numbers require IEEE-754 compliance
(`Require Import Floats`). Common sources of incompatibility
are checked at configure time, preventing compilation. In the
unlikely event an incompatibility remains undetected, using Floats
would enable proving False on this architecture.

Reply:
Both done thanks.